tools/build.py

```diff
@@ -244,11 +248,19 @@ def configure_jerry(arguments):
 
     return subprocess.call(cmake_cmd)
 
-def make_jerry(arguments, target=None):
-    make_cmd = ['make', '--no-print-directory', '-j', str(arguments.jobs), '-C', arguments.builddir]
+def make_jerry(arguments):
+    if sys.platform == 'win32':
+        make_cmd = ['cmake', '--build', arguments.builddir, '--config', arguments.build_type]
```

Comment:
Is the build type argument really needed? Shouldn't the cmake configure phase cache it?

Reply:
Yes, because vcproj files don't have default build type, you can build all build type with the same vcproj file. (if the only differences are compiler options and cmake project doesn't do any other trick)

Reply:
I see.